pdf/model/textencoding/glyphs_glyphlist.go

```diff
@@ -3,8576 +3,9105 @@
  * file 'LICENSE.md', which is part of this source code package.
  */
 /*
- * The embedded glyph to unicode mappings specified in this file are distributed under the terms listed in
- * ./glyphlist/glyphlist.txt.
+ * The embedded glyph to unicode mappings specified in this file are distributed under the terms
+ * listed in ./glyphlist/glyphlist.txt, ./glyphlist/texglyphlist.txt and ./glyphlist/addtional.txt
  */
 
 package textencoding
 
-var glyphlistGlyphToRuneMap = map[string]rune{
-	"A":                             '\u0041',
-	"AE":                            '\u00c6',
-	"AEacute":                       '\u01fc',
-	"AEmacron":                      '\u01e2',
+import (
+	"regexp"
+	"strconv"
+)
+
+// MissingCodeRune is the rune returned when there is no matching glyph. It was previously '?'.
+const MissingCodeRune = '\ufffd' // �
+
+// GlyphToRune returns true if `glyph` is in our GlyphToRune mapping.
+func KnownGlyph(glyph string) bool {
+	_, ok := GlyphToRune(glyph)
+	return ok
+}
+
+// GlyphToRune returns the rune corresponding to glyph `glyph` if there is one.
+// XXX: TODO: Can we return a string here? e.g. When we are extracting text, we want to get "ffi"
+//            rather than 'ﬃ'. We only need a glyph ➞ rune map when we need to convert back to
+//            glyphs.
+//            We are currently applying RuneToString to the output of functions that call
+//            GlyphToRune. While this gives the same result, it makes the calling code complex and
+//            fragile.
+// XXX: TODO: Can we combine all the tables glyphAliases, glyphlistGlyphToRuneMap,
```

Comment:
Combining would sacrifice explicitness, but maybe slightly more performant?  Also if there are any duplicate entries, then order matters.  Should be fine for now.